howto/send-events-with-output-bindings/README.md

```diff
@@ -42,5 +42,11 @@ curl -X POST -H  http://localhost:3500/v1.0/bindings/myEvent -d '{ "data": { "me
 ```
 
 As seen above, we invoked the `/binding` endpoint with the name of the binding to invoke, in our case its `myEvent`.
+The payload goes inside the mandatory `data` field, and can be any JSON serializable value.
 
-The payload goes inside the `data` field.
\ No newline at end of file
+
+## References
+
+* Binding [API](https://github.com/dapr/docs/blob/master/reference/api/bindings.md)
```

Comment:
Docs repo had some minor restructuring done during last release cycle, could you please update the links to new lo9cation.